hack/sandboxed.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:alpine as builder
+
+WORKDIR /go/src/sigs.k8s.io/krew
+
+# build binary
```

Comment:
I think we really shouldn't repeat the build process here. I highly recommend we just refactor the script that starts the container to do something like:

    OSARCH=linux/amd64 hack/make-binaries.sh
    docker run --rm -i -t -v ./out/bin/krew-linux_amd64:/bin/krew

Reply:
Hmm.. that will need a lot of extra work. I think I will remove the docker sandbox instead and submit that as a separate PR when `gox` is gone.